README.md

```diff
@@ -90,17 +72,6 @@ And/Or:
 
 Good luck and let the chitter begin!
 
-Code Review
------------
-
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want at this moment.
-
 Automated Tests:
 -----
 
```

Comment:
Maybe add a tiny bit of a plan to outline the steps you took? Or include the user stories and say which you did/ how you did them?